chain-storage/src/store.rs

```diff
@@ -300,7 +300,7 @@ fn compute_fast_link(depth: u64) -> u64 {
     }
 }
 
-//#[cfg(test)]
```

Comment:
Elsewhere we put the public test API under mod `testing`, distinct from mod `tests` which is conventional for the unit tests. let's follow the same convention.